app/controllers/experiments_controller.rb

```diff
@@ -95,6 +95,7 @@ def table
     @current_sort = view_state.state.dig('my_modules', view_mode, 'sort') || 'atoz'
 
     @project = @experiment.project
+    @archived_modules = @experiment.my_modules.archived.order(:name)
```

Comment:
Why do tasks load twice?

Reply:
i needed access to only archived tasks in the table view, since it was displaying only active tasks even in the archived state

Reply:
i dont, it just was there by default, whenever we changed to archive mode, it displayed active tasks instead of archived ones, so i added archive modules variable, it gets used when were in archive mode, active_modules gets used when we're in active view mode

Reply:
Just have a `@my_modules` variable, and load the appropriate collection based on parameter, something like:
`@my_modules = @experiment.my_modules.public_send(params[:view_mode] == "archived" ? :archived : :active).order(:name)`